### DISCUSSION TITLE
vim as a built-in sysext image and remove it from the base image

### DISCUSSION THREAD

**Discussion by tormath1** [flatcar/Flatcar#1463] [General]
Hey folks, we recently got this PR: https://github.com/flatcar/sysext-bakery/pull/78 to add vim-huge support to Flatcar. What if we could seize this opportunity to entirely remove vim from Flatcar and to propose it as a built-in Sysext extension instead?

In the past, vim was adding many CVEs (e.g: https://www.flatcar.org/releases#release-3510.2.0) - I am, personally, not aware of any vim use cases for Flatcar and for reading purposes one can still use `less`, `head`, etc.

Feel free to react :-1: or :+1: and / or give your point of view regarding this idea.

**Comment by ader1990** [comment]
Hello, I think we need an editor, being it either pico/nano or vi. Reading files and debugging is really hard without search or goto line number. I use vim all the time when running Flatcar, as it gives me freedom and fast lookup of things.

Also, the most important from my side: some files need to be modified at times, like /opt or various /etc configs. Doing it with `cat` is almost impossible in a decent manner. Maybe we can switch to nano or even a smaller vi implementation https://packages.gentoo.org/packages/app-editors/levee.
